src/store/vuex.ts

```diff
@@ -34,20 +33,25 @@ export class Store<
   A extends ActionsBase,
   M extends MutationsBase,
 > extends BaseStore<S> {
-  constructor(options: OriginalStoreOptions<S>) {
-    super(options);
+  constructor(options: StoreOptions<S, G, A, M>) {
+    super(options as OriginalStoreOptions<S>);
+    // @ts-expect-error Storeの型を書き換えている影響で未初期化として判定される
+    this.actions = dotNotationDispatchProxy(this.dispatch.bind(this));
+    this.mutations = dotNotationCommitProxy(
+      // @ts-expect-error Storeの型を書き換えている影響で未初期化として判定される
+      this.commit.bind(this) as Commit<M>,
+    );
   }
 
   readonly getters!: G;
 
-  // 既に型がつけられているものを上書きすることになるので、TS2564を吐く、それの回避
-
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error Storeの型を非互換な型で書き換えているためエラー
   dispatch: Dispatch<A>;
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
+  // @ts-expect-error Storeの型を非互換な型で書き換えているためエラー
   commit: Commit<M>;
+
+  actions: DotNotationDispatch<A>;
+  mutations: DotNotationCommit<M>;
```

Comment:
`.vue`とかで`store.actions`にカーソル当てたりコードジャンプすると出てくるのはこの辺りでしょうか。
簡単にメモ書きと工事中の案内を書いてあげると混乱する時間を減らせるかもと思いました！
こんな感じとか？

```suggestion
  /**
   * ドット記法用のActionを直接呼べる。エラーになる場合はdispatchを使ってください。
   * 詳細 https://github.com/VOICEVOX/voicevox/issues/2088
   */
  actions: DotNotationDispatch<A>;
  /**
   * ドット記法用のMutationを直接呼べる。エラーになる場合はcommitを使う。
   * 詳細 https://github.com/VOICEVOX/voicevox/issues/2088
   */
  mutations: DotNotationCommit<M>;
```